karapace/schema_reader.py

```diff
@@ -247,26 +249,29 @@ def handle_messages(self) -> None:
                 try:
                     key = ujson.loads(msg.key.decode("utf8"))
                 except ValueError:
-                    self._log.exception("Invalid JSON in msg.key: %r, value: %r", msg.key, msg.value)
+                    self._log.exception("Invalid JSON in msg.key")
```

Comment:
Is the idea to hide the values or to have less logging?

If the latter, would it be too much to log `msg.key`. Now this message has no context of what's wrong. We could always truncate the key to e.g. 16 characters.

Reply:
> Is the idea to hide the values or to have less logging?

I removed anything that is private data from the default log level, also the data you asked for is in the next line when verbose logging is used.

> Now this message has no context of what's wrong

That is what the exception message is for:

```
>>> import ujson
>>> import logging
>>> try:
...   ujson.loads('this is a test')
... except:
...   logging.exception('invalid json')
... 
ERROR:root:invalid json
Traceback (most recent call last):
  File "<stdin>", line 2, in <module>
ValueError: Unexpected character found when decoding 'true'
```